holoviews/plotting/widgets/__init__.py

```diff
@@ -206,7 +227,6 @@ def update(self, key):
         else:
             self.plot.update(key)
             self.plot.push()
-        return 'Complete'
```

Comment:
Wasn't this an ad hoc 'message' that meant something?

Reply:
It used to be the ACK for kernel.execute, that's now done via a comm.

Reply:
Ok. Glad to see it go.